### PR TITLE
Update binutils 2.28 for AT 11.0.

### PR DESCRIPTION
**Pull request by racardoso**
Updates binutils 2.28 to rev f85ae672d555. This fix a binutils
build error due to dynamic relocs (issue #19). Also includes
a patch to fix a incorrect pointer dereference at stabs.c.

Signed-off-by: Rogerio Alves Cardoso <rogealve@br.ibm.com>

### CODE REVIEW

**Comment by racardoso** [comment]
@rzinsly check fails for Ubuntu 16. It's 16.04?

**Comment by rzinsly** [comment]
@racardoso yes, failed for ck_requires. Its a known failure and its not related to your patch.

File: configs/11.0/packages/binutils/sources [hunk 1]
```diff
@@ -42,3 +42,20 @@ ATSRC_PACKAGE_PATCHES=
...
+		https://sourceware.org/ml/binutils/2017-02/msg00269.html \
+		fix-incorrect-indirection.patch 14 \
+		80e3c4062604f28f186289db0167b592 || return ${?}
+}
+
+atsrc_apply_patches ()
+{
+	#Fix error: comparison between pointer and zero character constant
+	#[-Werror=pointer-compare] in stabs.c
+	patch -p1 < fix-incorrect-indirection.patch
```
Comment (tuliom) [inline configs/11.0/packages/binutils/sources]: If this patch fails to apply, this function will still return 0.

Comment (racardoso) [inline configs/11.0/packages/binutils/sources]: @tuliom should `return ${?}` instead? There's other patches who do the same https://github.com/advancetoolchain/advance-toolchain/blob/master/configs/8.0/packages/binutils/sources#L58

Comment (tuliom) [inline configs/11.0/packages/binutils/sources]: @racardoso, yes.  But I prefer the solution you adopted in the previous function using ` || return ${?}`.
That way you prevent to add a bug if someone else adds a new command between `patch` and `return`.

Comment (racardoso) [inline configs/11.0/packages/binutils/sources]: @tuliom like ` patch -p1 < fix-incorrect-indirection.patch || return ${?}`? What about others places like 8.0? We should fix?

Comment (tuliom) [inline configs/11.0/packages/binutils/sources]: Yes, like that.
I think we should fix the other places, but I don't think that's a requirement to get your patch accepted.

Comment (racardoso) [inline configs/11.0/packages/binutils/sources]: @tuliom done.

**Pull request merged**